supabase/functions/manage-team-member/index.ts

```
@@ -0,0 +1,212 @@
+          return new Response(
+            JSON.stringify({ error: 'member_id and new_role are required for role updates' }),
+            { status: 400, headers: { ...corsHeaders, 'Content-Type': 'application/json' } }
+          )
+        }
+
+        const { data: roleUpdate, error: roleError } = await supabaseClient
+          .from('team_members')
+          .update({ role: new_role })
+          .eq('id', member_id)
+          .eq('team_id', team_id)
+          .select()
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)